src/components/LoginForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const LoginForm = () => {
+  const [email, setemail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post('http://localhost:8081/user/login', { email, password });
+      // Assuming backend responds with a token upon successful login
+      const token = response.data.token;
+      // Store token in local storage or state for future requests
+      localStorage.setItem('token', token);
+      // Redirect or perform other actions upon successful login
+    } catch (error) {
```

Comment:
Log this error

Reply:
![Screenshot (727)](https://github.com/learning-project-01/bookstore-view/assets/123824203/4b3e5b4e-a94e-489e-8051-ef84497b15e5)